websites/P/poke/metadata.json

```diff
@@ -0,0 +1,25 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.10",
+	"author": {
+		"name": "ashley",
+		"id": "396571938081865741"
+	},
+	"service": "Poke",
+	"description": {
+		"en": "Watch Silly videos or search the interwebs while being anonymous on poke!"
+	},
+	"url": [
+		"poketube.fun",
+		"poke.ggtyler.dev",
+		"poke.vern.cc"
+	],
+	"version": "1.0.0",
+	"logo": "https://cdn.glitch.global/d68d17bb-f2c0-4bc3-993f-50902734f652/yt-ukraine.png",
```

Comment:
[Logos](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md#logo) must be 512x512px